Assets/Scripts/UI/PaletteSwap.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEditor;
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    public class PaletteSwap : MonoBehaviour
+    {
+        private Image image;
+        // [SerializeField] private List<Color> colors;
+        [SerializeField] private ColorPalette palette;
+        [SerializeField] private bool allowRuntimeUpdateChange;
+
+        private static readonly int PaletteTex = Shader.PropertyToID("_PaletteTex");
+
+        private void Awake()
+        {
+            UpdatePalette();
```

Comment:
Ik mis dat de image hier geset wordt; OnValidate wordt alleen in de editor aangeroepen en dan zou in de build de Update stuk gaan omdat de image nog niet geset is

Reply:
goeie, had ik over het hoofd gezien